src/util/api.js

```diff
@@ -98,3 +98,15 @@ export const initiatePrivileged = body => {
 export const transitionPrivileged = body => {
   return post('/api/transition-privileged', body);
 };
+
+// Create user with identity provider (e.g. Facebook or Google)
+//
+// If the authentication with idp fails, we will create a new user.
+// For that user needs to confirm data fetched from the idp.
+// This endpoint creates the new user with confirmed data.
+//
+// See `server/api/auth/createWithIdp.js` to see what data should
+// be sent in the body.
+export const createWithIdp = body => {
+  return post('/api/auth/create-with-idp', body);
+};
```

Comment:
Sorry, this is not really related to this PR, but I feel that this util/api.js is pretty confusing file due to naming. 

Could we add some description to the beginning of the file (or somewhere) that these helpers are about client app's server-side routes (not something that is directly calling Marketplace API and Integration API)? 

When we mention API in Flex Docs / Slack / Intercom, it's pretty much always about those Flex APIs instead of these client app's own server routes (which currently are a kind of "proxy" calls, but that's likely to change). So, this file might cause some confusion without better explanation somewhere.